### ISSUE TITLE
No skov.image

### ISSUE THREAD

**Issue by no-e-in** [nicolas-p/skov#14]
In windows 7 with factor-windows-x86-64-0.98, following the procedure in the file README.md, the skov.image is not generated.

**Comment by nicolas-p** [comment]
Could you post a screenshot (or file listing) of the`factor` folder (containing skov.exe or factor.exe) after the build process crashes?

**Comment by no-e-in** [comment]
After make-skov.factor, skov.exe is generated but not [skov.image](https://ibb.co/hy3sRCm). By trying to enter the procedures contained in make-skov. factor directly into the Listener, the process abort to  ["code.import-export -> Vocabolary does not exist"](https://ibb.co/R2sKn2R).

**Comment by nicolas-p** [comment]
Solved in da0c6824d3fbec4bbf4a4beedb9208cb39cff76e

The problem was that in `make-skov.factor` I was running a loop on all the words in the system to rename some of them, and I accidentally removed the last `drop` two years ago, which means that the data stack was growing bigger at every iteration. For reasons that I don't understand it was not causing any crash on my Mac but it was causing a "data stack overflow" error on Windows. It could be due to a difference between the Mac and Windows versions of Factor.

I also made some changes today to make the Windows version actually usable. Because of some bug(s) in the Windows version of Factor, it is impossible to display text on a gradient background, so by default every label is written black on black. I am now displaying a flat colour on Windows. Also antialiasing doesn't seem to work on Windows, and some characters do not display properly.

So you will see that Skov still looks awful on Windows but at least it runs now.
